src/sync/index.js

```diff
@@ -413,7 +415,7 @@ async function sync(db){
 async function updateOraclesPassedEndBlock(currentBlockChainHeight, db, resolve){
   // all central & decentral oracles with VOTING status and endBlock less than currentBlockChainHeight
   try {
-    await db.Oracles.findAndModify({endBlock: {$lt:currentBlockChainHeight}, status: 'VOTING'}, [], {$set: {status:'WAITRESULT'}}, {});
+    await db.Oracles.update({endBlock: {$lt:currentBlockChainHeight}, status: 'VOTING'}, {$set: {status:'WAITRESULT'}}, {multi: true});
```

Comment:
why does this one have {multi: true} and some others dont?

Reply:
without `{multi: true}`, only one row returned from query will be updated